hypothesis-python/src/hypothesis/core.py

```diff
@@ -529,6 +529,7 @@ def __init__(
 
         self.files_to_propagate = set()
         self.failed_normally = False
+        self.failed_over_deadline = False
```

Comment:
Nit: It took me a while to figure out what this variable name is trying to communicate. Perhaps something like `failed_due_to_deadline` would be clearer?